scranShipper/src/DisplayHomepage/index.jsx

```diff
@@ -0,0 +1,27 @@
+const DisplayHomepage = () => {
+
+    return (
+        <>
+            <div className="container shadow">
+                <h1 className="col-12 text-center p-3 text-sm-start fs-3"><em className="text-primary">Scran</em>Shippers
+                </h1>
+            </div>
+            <section className="container py-5 px-3" style={{
+                backgroundImage: 'url("/public/hamburger.webp")',
+                backgroundRepeat: "no-repeat",
+                backgroundSize: "cover",
+                backgroundPosition: "top right"
+            }}>
```

Comment:
Rather than adding inline css here, make a `DisplayHomepage.css` with the custom css and import it.